Mathlib/Algebra/Polynomial/RingDivision.lean

```diff
@@ -234,7 +234,6 @@ theorem rootMultiplicity_mul {p q : R[X]} {x : R} (hpq : p * q ≠ 0) :
     multiplicity_mul (prime_X_sub_C x) (finiteMultiplicity_X_sub_C _ hpq)]
 
 open Multiset in
-set_option linter.unusedVariables false in
```

Comment:
I guess it's not supposed to be included. But it's a good change, so I'm not going to block over this. And writing this comment was also a bit of a waste of time probably :)